src/FortTroopCalculator.tsx

```diff
@@ -0,0 +1,394 @@
+    infTroop: 0,
+    rangeTroop: 0,
+    cavTroop: 0,
+    totalTroops: 200000,
+    t5Troops: 60,
+  });
+  const [result, setResult] = React.useState({
+    t5inf: 0,
+    t5range: 0,
+    t5cav: 0,
+    t4inf: 0,
+    t4range: 0,
+    t4cav: 0,
+  });
+  const calculateTroops = () => {
+    const troopsCalculated = troopCalculator(troops);
+    setResult({
+      t5inf: troopsCalculated.t5.t5inf,
+      t5range: troopsCalculated.t5.t5range,
+      t5cav: troopsCalculated.t5.t5cav,
+      t4inf: troopsCalculated.t4.t4inf,
+      t4range: troopsCalculated.t4.t4range,
+      t4cav: troopsCalculated.t4.t4cav,
+    });
+  };
+  const totalComp = troops.infTroop + troops.rangeTroop + troops.cavTroop;
+
+  return (
+    <div className="mt-10 sm:mt-0">
+      <div className="md:grid md:grid-cols-3 md:gap-6">
+        <div className="md:col-span-1">
+          <div className="px-4 sm:px-0">
+            <h2 className="text-3xl font-bold tracking-tight text-gray-900">
+              Troop Calculator for Forts
+            </h2>
+
+            <p className="mt-4 text-lg text-gray-500">
+              Enter the troop Comp , troops and t5 percent you want to
+              calculate!
+            </p>
+          </div>
+        </div>
+        <div className="mt-5 md:col-span-2 md:mt-0">
+          <div className="overflow-hidden shadow sm:rounded-md">
+            <div className="bg-slate-100 px-4 py-5 sm:p-6">
+              <div className="grid grid-cols-9 gap-6">
+                <div className="col-span-4 sm:col-span-3">
+                  <label
+                    htmlFor="infTroop"
+                    className="block text-sm font-medium text-gray-700"
+                  >
+                    Infantry
+                  </label>
+                  <select
+                    id="infTroop"
+                    name="infTroop"
+                    onChange={(e) =>
+                      setTroops({
+                        ...troops,
+                        infTroop: parseFloat(e.target.value),
+                      })
+                    }
+                    value={troops.infTroop}
+                    className="mt-1 block w-full rounded-md border border-gray-300 bg-white py-2 px-3 shadow-sm focus:border-sky-500 focus:outline-none focus:ring-sky-500 sm:text-sm"
+                  >
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                  </select>
+                </div>
+                <div className="col-span-4 sm:col-span-3">
```

Comment:
you can probably create a component to reuse this no? Take as props the label the key etc. Only that changes the rest are copied between the 3 selects

Reply:
maybe we can leave this as a new issue since its creating a new component and its a bigger change and make a new pr for it maybe do the same on the results component as well? what do you say?